pkg/actions/lua.go

```diff
@@ -128,6 +133,34 @@ func LuaRun(l *lua.State, code, name string) error {
 	return l.ProtectedCall(0, lua.MultipleReturns, 0)
 }
 
+func (h *LuaHook) collectStats(l *lua.State) {
+	// TODO(isan) how to configure?
+	packagesToReport := map[string]bool{
+		"lakefs/catalogexport/hive":             true,
+		"lakefs/catalogexport/symlink_exporter": true,
+		"lakefs/catalogexport/glue_exporter":    true,
+	}
+
+	if l.IsTable(lua.RegistryIndex) {
+		l.Field(lua.RegistryIndex, "_LOADED")
+		l.PushNil()
+		for l.Next(-2) {
+			// Key is at index -2, value is at index -1
+			key := lua.CheckString(l, -2)
+			t := l.TypeOf(-1)
+			// value := l.ToValue(-1)
+			if t == lua.TypeTable {
+				if packagesToReport[key] {
+					h.stats.CollectEvent(stats.Event{Class: "lua_hook_run", Name: key})
```

Comment:
discuss with @NoyDavidson2 about the new metric report

Reply:
Good idea, meeting with @NoyDavidson2 later today and will make sure before merge also will let you know if anything changes after speaking with her 👍

Reply:
@nopcoder after speaking with @NoyDavidson2 it's now `lua_hooks` - the only update required since your review.